pkg/util/json/json.go

```diff
@@ -442,6 +447,99 @@ func (j jsonObject) EncodeInvertedIndexKeys(b []byte) [][]byte {
 	return outKeys
 }
 
+// MakeJSONFromDecimal returns a JSON value given a apd.Decimal.
+func MakeJSONFromDecimal(v apd.Decimal) JSON {
+	return jsonNumber(v)
+}
+
+// MakeJSONFromArrayOfJSON returns a JSON value given a []JSON.
+func MakeJSONFromArrayOfJSON(v []JSON) JSON {
+	return jsonArray(v)
+}
+
+// MakeJSONFromNumber returns a JSON value given a json.Number.
+func MakeJSONFromNumber(v json.Number) (JSON, error) {
+	// The JSON decoder has already verified that the string `v` represents a
+	// valid JSON number, and the set of valid JSON numbers is a [proper] subset
+	// of the set of valid apd.Decimal values.
+	dec := apd.Decimal{}
+	_, _, err := dec.SetString(string(v))
+	return jsonNumber(dec), err
+}
+
+// MakeJSONFromString returns a JSON value given a string.
+func MakeJSONFromString(v string) JSON {
+	return jsonString(v)
+}
+
+// MakeJSONFromBool returns a JSON value given a bool.
+func MakeJSONFromBool(v bool) JSON {
+	if v {
+		return TrueJSONValue
+	}
+	return FalseJSONValue
+}
+
+// makeJSONFromArrayOfInterface returns a JSON value given a []interface{}.
+func makeJSONFromArrayOfInterface(v []interface{}) (JSON, error) {
```

Comment:
Since this function is unexported I think we can get away with a slightly less verbose name, maybe just `fromArray`?